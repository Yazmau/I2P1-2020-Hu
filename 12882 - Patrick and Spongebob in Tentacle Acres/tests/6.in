15 37
56 76 79 32 58 92 65 39 35 81 30 71 80 58 91 94 54 51 69 100 78 1 45 26 2 76 32 51 57 67 36 51 71 75 86 18 90
45 82 90 14 51 22 84 94 95 30 99 49 36 13 37 18 47 40 37 89 63 94 47 51 25 90 7 46 49 56 98 23 71 22 7 41 31
33 100 65 2 62 78 84 79 2 57 7 18 74 93 28 17 76 30 18 57 88 9 61 69 47 13 100 68 35 67 65 90 9 26 16 88 30
68 56 32 79 42 40 44 90 16 84 20 14 39 9 88 42 98 5 16 7 53 49 43 37 9 20 87 58 98 71 3 99 11 16 12 49 78
77 5 13 17 19 74 85 46 41 9 77 20 41 52 38 56 82 17 78 97 12 30 82 40 23 49 91 73 76 26 18 20 63 79 13 32 47
87 11 60 56 20 87 1 9 75 38 73 44 92 68 95 85 68 95 13 38 2 84 41 88 42 8 96 25 12 13 18 4 17 22 80 4 65
74 76 5 24 82 1 50 14 38 88 57 28 88 48 70 8 53 67 84 24 22 59 55 91 73 66 40 34 81 86 8 97 61 62 54 80 10
47 8 55 60 39 18 34 50 57 82 25 7 40 9 92 69 100 69 81 61 84 53 69 81 30 21 31 5 93 11 41 95 86 38 5 71 24
39 14 63 93 43 52 20 74 82 11 51 55 16 24 74 64 18 87 73 21 37 11 13 95 72 61 47 85 69 86 30 4 32 44 41 95 44
49 90 19 39 4 2 40 78 2 79 3 91 54 84 66 94 29 25 95 33 28 40 69 28 21 69 22 49 29 75 83 61 33 88 11 89 19
81 100 9 78 72 44 85 36 77 21 51 58 33 42 11 48 58 68 63 58 24 49 67 88 21 13 98 94 83 20 95 97 29 17 77 40 65
7 15 52 49 74 12 69 17 86 37 19 87 1 98 73 36 72 69 69 28 49 51 10 37 52 81 45 100 25 96 73 7 4 17 27 98 74
86 70 94 17 95 12 37 24 18 67 11 44 27 45 3 52 45 37 56 70 87 75 32 100 60 100 5 36 47 47 43 100 16 49 99 82 29
19 19 29 94 99 11 87 45 70 38 73 83 93 51 38 41 99 65 88 64 77 19 18 90 22 84 8 46 47 80 50 96 96 33 33 26 21
28 36 12 22 77 60 13 100 58 61 52 2 49 63 12 28 1 58 33 59 69 10 72 19 54 87 87 44 60 11 2 18 28 39 75 89 57
58
5 7 31 31
10 13 15 27
11 12 2 5
10 13 36 36
4 4 27 37
2 9 14 24
4 15 22 37
9 15 27 30
1 2 31 37
6 8 20 32
15 15 35 35
14 15 33 34
8 8 10 22
14 15 20 22
15 15 14 25
6 11 11 36
13 14 31 33
12 14 37 37
11 13 31 32
11 11 35 36
2 3 19 21
10 11 15 21
4 15 25 29
10 12 8 33
13 14 4 32
1 4 30 34
10 15 36 37
3 12 18 19
10 12 29 37
9 15 2 22
8 13 33 37
6 13 9 15
15 15 5 35
6 9 21 32
7 15 33 33
12 13 16 23
11 12 13 27
14 15 25 36
14 15 27 32
8 8 31 31
15 15 34 36
4 11 36 36
7 9 23 35
15 15 19 34
13 13 16 25
7 7 22 35
4 10 30 30
1 10 35 37
12 13 19 21
10 11 3 18
2 5 30 30
1 15 24 29
14 14 30 31
15 15 9 20
9 10 19 37
12 14 5 12
5 6 12 35
14 14 34 36
