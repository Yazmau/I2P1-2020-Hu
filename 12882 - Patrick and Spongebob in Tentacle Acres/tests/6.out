18
100
100
98
99
100
100
100
98
97
75
96
100
77
72
100
100
74
100
77
89
95
100
100
100
99
98
95
98
100
99
98
100
97
96
87
98
96
96
41
89
95
97
87
100
97
86
95
87
94
71
100
80
63
95
99
97
33
